Servers/WebServer/Client.cpp

```diff
@@ -139,15 +139,15 @@ void Client::send_response(StringView response, const Core::HttpRequest& request
 void Client::send_redirect(StringView redirect_path, const Core::HttpRequest& request)
 {
     StringBuilder builder;
-    builder.append("HTTP/1.0 301 Moved Permanently\r\n");
+    builder.append("HTTP/1.0 302 Found\r\n");
```

Comment:
No, it actually is a permanent redirect. Unless the directory is removed and a file is created in its place, but by that criteria, nothing's permanent.

Reply:
Ok, will leave it as a 301